marconi-chain-index/src/Marconi/ChainIndex/Experimental/Runner.hs

```diff
@@ -17,24 +16,32 @@ import Cardano.Api.Extended.Streaming (
 import Cardano.BM.Data.Trace (Trace)
 import Cardano.BM.Trace qualified as Trace
 import Control.Concurrent qualified as Concurrent
+import Control.Concurrent.Async qualified as Async
 import Control.Concurrent.STM qualified as STM
 import Control.Exception (catch)
 import Control.Monad.Except (ExceptT, void)
-
 import Data.Text (Text)
-import Data.Void (Void)
 import Marconi.ChainIndex.Experimental.Extract.WithDistance (WithDistance)
 import Marconi.ChainIndex.Experimental.Extract.WithDistance qualified as Distance
 import Marconi.ChainIndex.Experimental.Indexers.Orphans qualified ()
 import Marconi.ChainIndex.Logging (chainSyncEventStreamLogging)
 import Marconi.ChainIndex.Node.Client.Retry (RetryConfig, withNodeConnectRetry)
-import Marconi.ChainIndex.Utils qualified as Utils
+import Marconi.ChainIndex.Types (SecurityParam)
 import Marconi.Core.Experiment qualified as Core
-import Prettyprinter qualified as Pretty
-import Prettyprinter.Render.Text qualified as Pretty
+import Prettyprinter qualified as PP
+import Prettyprinter.Render.Text qualified as PP
 import Streaming qualified as S
 import Streaming.Prelude qualified as S
 
+data RunIndexerLog
```

Comment:
Add comment before the merge.

Reply:
1ded0f7b06623d1f61ad98b08f34bc4993f74048